docker/README_DEV.md

```diff
@@ -19,33 +19,80 @@ Once the image is built it needs to be tagged as `eclipse/rdf4j-workbench:${mave
 
 The simplest way to accomplish all of the above is to run the `build.sh` script.
 
-## Running up the docker container
+### Running up the docker container (docker-compose)
 
 Use docker-compose to run up the container. This uses the `docker-compose.yml` file to start
 a container with the port mapped to 8080 and persistence using docker volumes.
 
 `docker-compose up -d`
 
-### Other useful commands
+#### Other useful docker-compose commands
+
+Stop:
 
-#### Stop
 `docker-compose stop`
 
-#### Follow logs
+Follow logs:
+
 `docker-compose logs -f`
 
-#### Show all logs
+Show all logs:
+
 `docker-compose logs --tail="all"`
 
-#### Start containers in foreground (prints logs and stops containers with Ctrl-C)
+Start containers in foreground (prints logs and stops containers with Ctrl-C):
+
 `docker-compose up`
 
-#### Stop and remove all containers and delete all my volumes (for all containers and all volumes)
+Stop and remove all containers and delete all my volumes (for all containers and all volumes):
+
 `docker stop $(docker ps -a -q); docker rm $(docker ps -a -q); docker system prune -f --volumes`
 
-## Deploy
+List all versions on the local machine:
+
+`docker images | grep "eclipse/rdf4j-workbench"`
+
+## Deploy the image on hub.docker.com
+
+The docker images on hub.docker.com are stored as part of the Eclipse organizational account. 
+
+Since this account is managed separately by the Eclipse Foundation,
+only a limited number of committers will be granted access by the EMO.
+
+### Method 1: using the build script and docker push
+
+Build the SDK ZIP file and docker image using the `build.sh` script mentioned above.
+Both the Workbench and the server will be part of the same image.
+
+Log into hub.docker.com:
+
+`docker login --username=yourhubusername`
+
+Push the image:
+
+`docker push eclipse/rdf4j-workbench:VERSION_TAG`
+ 
+`VERSION_TAG` is the version (tag) you want to push, e.g. `4.3.0`
```

Comment:
How do we replace the version tagged with 'latest' ? Or does that happen automagically?

Reply:
Build script will pick up the version number from the main pom.xml file and use it to tag your docker image.

Reply:
Yes, tagging and pushing 'latest' needs to be done explicitly (either manually or in a script).

Reply:
Cool, makes sense. Just because I know myself and will forget this again: can we add a sentence to clarify that you _should_ also push the `latest` tag explicitly (in addition to the actual version tag)?

Reply:
Added a note about 'latest'